Sources/TuistCache/Cache/CacheRemoteStorage.swift

```diff
@@ -151,6 +151,8 @@ public final class CacheRemoteStorage: CacheStoring {
             return xcframeworkPath
         } else if let frameworkPath = FileHandler.shared.glob(archive, glob: "*.framework").first {
             return frameworkPath
+        } else if let bundlePath = FileHandler.shared.glob(archive, glob: "*.bundle").first {
```

Comment:
Nitpick: I'd rename this method then to something like `artifactPath`. A bundle is not a framework :)